libs/designsystem/src/lib/helpers/unique-id-generator.helper.ts

```diff
@@ -0,0 +1,19 @@
+export class UniqueIdGenerator {
+  private static registry = new Map<string, UniqueIdGenerator>();
+  private id = 0;
+
+  private constructor(private scope: string) {}
+
+  static scopedTo(scope: string) {
```

Comment:
```suggestion
  static getScopedGenerator(scope: string) {
```

Reply:
@lydemann You have to see it in context:
`UniqueIdGenerator.scopedTo('my-scope');`